src/components/DocSigningField.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<p>
+		<label :for="id">
+			{{ label }}
+		</label>
+		<br>
+		<textarea :id="id"
+			v-model="localValue"
+			:name="name" />
+		<br>
+		<button :id="id + 'Save'" @click="onClickSave">
+			<span :title="saveTooltip" data-toggle="tooltip">
+				Save
+			</span>
+		</button>
```

Comment:
Can you switch that to use the [NcButton](https://nextcloud-vue-components.netlify.app/#/Components/NcButton) component? We can drop the tooltip as we have a title on the button, use the `t()` function to make text translatable
```suggestion
		<NcButton @click="onClickSave" type="secondary">
			{{ t('richdocuments', 'Save') }}
		</NcButton>
```

Reply:
Done